docs/contributing.md

```diff
@@ -32,6 +32,14 @@ If you want to contribute and don't know where to start, here is a step-by-step
 7. Run `pnpm migrations migrate` - apply database migrations
 8. Run `pnpm dev` - URL will be logged
 
+## Creating database
+
+If you are configuring the app for the first time. Create the database before running migrations.
+
+```sh
+user=# CREATE DATABASE webstudio;
```

Comment:
what this mean?

Reply:
Removed it, was just a random path in the shell